frontend/smart-kickers-game/src/App.js

```diff
@@ -22,6 +22,12 @@ function App() {
     };
   }, []);
 
+  function handleResetGame() {
+    resetGame().then((data) => {
+      alert(data.error);
+    });
+  }
```

Comment:
This code will either show (`data.error.message` anybody?) if the request fail:
<img width="468" alt="image" src="https://user-images.githubusercontent.com/10504661/182823414-b6281c30-b4da-42c4-ad9d-f54eab7d9810.png">
or this, if the request succeeds:
<img width="464" alt="image" src="https://user-images.githubusercontent.com/10504661/182823644-0a54672a-44c6-4c24-bd1c-697e920fc620.png">

Reply:
My fault. 
Now it will only show when the error occurred